[Bulk] Broken Link fixes (#878)
MicrosoftDocs/biztalk-docs: biztalk/technical-guides/checklist-getting-started-with-biztalk-server.md

|                                                                                                                                                                                                                 Learn about the skill set required for using [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)].                                                                                                                                                                                                                 |                                                                                                                                                                                                                                                   See [Prerequisite Skills and Knowledge](../core/prerequisite-skills-and-knowledge5.md).                                                                                                                                                                                                                                                   |

> |                                                                                                                                                                                                                                                            Understand and read about BizTalk Server.                                                                                                                                                                                                                                                             |                                                                                                                                                                 -   See [Introducing BizTalk Server 2010](../core/introducing-biztalk-server.md).<br />-   See [Microsoft BizTalk Server 2010 Technical Overview](https://go.microsoft.com/fwlink/?LinkId=202654).                                                                                                                                                                 |

Reviewer: FW lInk doesn't go to anything:

```suggestion
|                                                                                                                                                                                                                                                            Understand and read about BizTalk Server.                                                                                                                                                                                                                                                             |                                                                                                                                                                 -   See [Introducing BizTalk Server 2010](../core/introducing-biztalk-server.md) and [Microsoft BizTalk Server 2010 Technical Overview](/biztalk/technical-guides/biztalk-server-2010-technical-overview). |
```

> |                                                                                                                                                                      Look at [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] posters to understand the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] architecture.                                                                                                                                                                       |                                                                                                                                                                                                                                                        See [BizTalk Server Posters](https://go.microsoft.com/fwlink/?LinkId=155865).                                                                                                                                                                                                                                                         |

Reviewer: FW link doesn't to anything:

```suggestion
|                                                                                                                                                                      Look at [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] posters to understand the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] architecture.                                                                                                                                                                       |                                                                                                                                                                                                                                                        See [BizTalk Server Posters](https://download.microsoft.com/download/E/8/B/E8B0CAD0-5A1D-4A1A-A2DC-5C9962C76AC5/BizTalk_Server_2010_Posters.pdf).                                                                                                                                                                                                                                                         |
```

|                                                                                                                                               Read through [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] case studies to understand how organizations deploy [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] to meet their business goals.                                                                                                                                               |                                                                                                                                                                                                                                       See [Security Case Studies for Small & Medium-Sized Companies](../core/security-case-studies-for-small-to-medium-sized-companies.md).                                                                                                                                                                                                                                        |

> |                                                                                                                                Look at the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] Developer Center to access various [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] resources such as documentation, blogs, videos, webcasts, and announcements.                                                                                                                                 |                                                                                                                                                                                                                                                    See [BizTalk Server Developer Center](https://go.microsoft.com/fwlink/?LinkId=158758).                                                                                                                                                                                                                                                    |

Reviewer: BizTalk Server Developer Center doesn't exist:

```suggestion
```

| Install and configure BizTalk Server. As part of [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] installation and configuration, you must also configure Windows Server and SQL Server for optimum [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] performance.<br /><br /> If you are going to publish [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] Web services and WCF services, you must also configure the Internet Information Services (IIS). | -   See [BizTalk Server 2010 Installation Guides](https://go.microsoft.com/fwlink/?LinkId=183138).<br />-   See [Checklist: Configuring Windows Server](~/technical-guides/checklist-configuring-windows-server.md)<br />-   See [Checklist: Configuring Internet Information Services](~/technical-guides/checklist-configuring-internet-information-services.md)<br />-   See [Checklist: Configuring SQL Server](~/technical-guides/checklist-configuring-sql-server.md)<br />-   See [Checklist: Configuring BizTalk Server](~/technical-guides/checklist-configuring-biztalk-server.md) |

> |                                                                                                                                                                 Look at videos of [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] to learn how to perform key tasks using [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)].                                                                                                                                                                 |                                                                                                                                                                                                                                                         See [BizTalk Server Videos](https://go.microsoft.com/fwlink/?LinkId=155889).                                                                                                                                                                                                                                                         |

Reviewer: Gone:

```suggestion
```

|                                                                                                                                                    Download and use the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] samples to see how to create a [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] solution using the key features.                                                                                                                                                    |                                                                                                                                                                                                                                                    See [BizTalk Server 2010 SDK Samples](../core/samples-in-the-sdk.md).                                                                                                                                                                                                                                                    |

> |                                                                                                                                                                                       Refer to the BizTalk Server Help and other white papers to know how to perform specific tasks in [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)].                                                                                                                                                                                       |                                                                                                                                                      -   **Online Help** – [Microsoft BizTalk Server 2010 Help](../core/biztalk-server-core-documentation.md).<br />-   **Downloadable Help** – [BizTalk Server 2010 Documentation](https://go.microsoft.com/fwlink/?LinkId=202656).                                                                                                                                                      |

Reviewer: BizTalk 2010 online help download is gone:

```suggestion
|                                                                                                                                                                                       Refer to the BizTalk Server Help and other white papers to know how to perform specific tasks in [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)].                                                                                                                                                                                       |                                                                                                                                                      -   **Online Help** – [Microsoft BizTalk Server Help](../core/biztalk-server-core-documentation.md).                                                                                                                                                      |
```

|                                                                                                                                                                                                                          **Plug into the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] community**                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |

> |                                                                                                                                                                                                                               Get support for [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)].                                                                                                                                                                                                                                |                                                                                                                                                                                                                                                        See [BizTalk Server Support](https://go.microsoft.com/fwlink/?LinkId=156438).                                                                                                                                                                                                                                                         |

Reviewer: FW link doesn't go to BizTalk Server Support:

```suggestion
|                                                                                                                                                                                                                               Get support for [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)].                                                                                                                                                                                                                                |                                                                                                                                                                                                                                                        See [Microsoft Support](https://support.microsoft.com/contactus/).                                                                                                                                                                                                                                                         |
```

|                                                                                                                                                                                                                                                            Participate in BizTalk-focused newsgroups.                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                              See [Newsgroups](/previous-versions/aa937644(v=msdn.10)).                                                                                                                                                                                                                                                               |

> |                                                                                                                                                                                                                                                               Read through BizTalk-related blogs.                                                                                                                                                                                                                                                                |                                                                                                                                                                                                                                                          See BizTalk Server Blogs                                                                                                                                                                                                                                                           |

Reviewer: ```suggestion
|                                                                                                                                                                                                                                                               Read through BizTalk-related blogs.                                                                                                                                                                                                                                                                |                                                                                                                                                                                                                                                          See [BizTalk Server Team Blog](https://techcommunity.microsoft.com/t5/biztalk-server-team-blog/bg-p/BizTalkServerTeamBlog)                                                                                                                                                                                                                                             |
```

|                                                                                                                                                                                                                               **Latest about [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)]**                                                                                                                                                                                                                                |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |

> |                                                                                                                                                                                                                   Keep yourself updated with the latest on [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)].                                                                                                                                                                                                                   |                                                                                                                                                                                                                                                       See [Microsoft BizTalk Server](https://go.microsoft.com/fwlink/?LinkId=156443).                                                                                                                                                                                                                                                        |

Reviewer: Link doesn't go to anything related or useful:
```suggestion
```

## See Also

>  [Operational Readiness Checklists](~/technical-guides/operational-readiness-checklists.md)

Reviewer: ```suggestion
[Operational Readiness Checklists](~/technical-guides/operational-readiness-checklists.md)
```
